aws/structure.go

```diff
@@ -5565,6 +5565,23 @@ func flattenAppsyncPipelineConfig(c *appsync.PipelineConfig) []interface{} {
 	return []interface{}{m}
 }
 
+func flattenAppsyncCachingConfig(c *appsync.CachingConfig) []interface{} {
```

Comment:
Maybe move to `aws/resource_aws_appsync_resolver.go` to avoid overloading this file?

Reply:
That makes sense, thank you!

I'll also move `flattenAppsyncPipelineConfig` together with `flattenAppsyncCachingConfig ` to `aws/resource_aws_appsync_resolver.go`. 💡 

Btw, besides `flattenAppsync**`, I can see some resource-specific function as well in `structure.go` but I'm not sure why the functions are in the `structure.go`. 🤔